start.sh

```diff
@@ -0,0 +1,6 @@
+#!/usr/bin/env bash
+
+SOURCE=$(dirname ${BASH_SOURCE[0]})
+echo -n "source: "
+echo $SOURCE
+node "${SOURCE}/query-engine/driver-adapters/js/connector-test-kit-executor/dist/index.mjs"
```

Comment:
Given we are going to continue working on this, and that we would need a mechanism to configure the tests properly (which will be encoded in a GitHub action) I believe we should remove the two bash scripts above.